tests/test_create.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function
 
-import pytest
 from contextlib import contextmanager
 from glob import glob
 from logging import getLogger, Handler
 from os.path import exists, isdir, join, relpath
 from shlex import split
 from shutil import rmtree
+import subprocess
+import sys
 from tempfile import gettempdir
 from unittest import TestCase
 from uuid import uuid4
 
+from menuinst.win32 import dirs as win_locations
```

Comment:
RE rules of thumb, menuinst is only a dependency for conda on windows, right? So in that case it should only be imported if on windows, and an in-line import would ensure that's the case.